### ISSUE TITLE
RN: in-line image via markdown is not scaling well

### ISSUE THREAD

**Issue by dmarkdvt** [mattermost/mattermost-server#12970]
#### Summary
In the mobile app the inline image via markdown is not scaling well; px declaration is broken and svg can not be displayed. It is related to https://mattermost.atlassian.net/browse/MM-14628, which IMHO was closed wrongly.

#### Steps to reproduce
Mattermost Version: 5.16.1
Android Version: 1.24.0

Enter the following commands.
1: `![status](https://talk.vtirol.gv.at/DVTImages/jira-status/inprogress_s.png)`
2: `![status](https://talk.vtirol.gv.at/DVTImages/jira-status/inprogress.png)`
3: `![status](https://talk.vtirol.gv.at/DVTImages/jira-status/inprogress_b.png)`
4: `![status](https://api.travis-ci.org/mattermost/mattermost-server.svg)`
5: `![status](https://talk.vtirol.gv.at/DVTImages/jira-status/inprogress.png =50 "foobar")`
6: `![status](https://talk.vtirol.gv.at/DVTImages/jira-status/inprogress.png =50x50 "foobar")`

#### Expected behavior
1: The picture (32x8 px) not scaling properly to the text. It should be small but it is shown with a minimum height.

2: The picture (67x17 px) not scaling properly to the text. It is displayed as it is. It is even in-line (as recommended in https://mattermost.atlassian.net/browse/MM-14628) and not between a text. But regarding to the docu https://docs.mattermost.com/help/messaging/formatting-text.html#in-line-images that is what this tag is meant for?!

3: Same as 2 but with a bigger picture (300x76 px).

4: SVG is not working. Was just a test: https://mattermost.atlassian.net/browse/MM-9746

5: Should be displayed like in 6.

6: Is ignoring the size setting.

#### Observed behavior (that appears unintentional) 
Web app: ![image](https://user-images.githubusercontent.com/57185425/67869166-58987f80-fb2d-11e9-964c-5e6b63036df9.png)

Android app: 
![image](https://user-images.githubusercontent.com/57185425/67869514-d3619a80-fb2d-11e9-8d15-055396be3d7d.png)

#### Possible fixes
- allow css
- display svg in rn
- allow color
- allow more via latex

**Comment by amyblais** [comment]
@dmarkdvt We're unable to load the inprogress_s.png images that you linked. Would you be open to providing public images so we can try to reproduce?

**Comment by dmarkdvt** [comment]
@amyblais I just tested them via mobile connection and there shouldn't be any limitation.

https://talk.vtirol.gv.at/DVTImages/jira-status/inprogress_s.png
https://talk.vtirol.gv.at/DVTImages/jira-status/inprogress.png
https://talk.vtirol.gv.at/DVTImages/jira-status/inprogress_b.png

**Comment by amyblais** [comment]
An update on this is that we don’t support rendering svg’s on the mobile app. One of our devs was looking into this at some point and shared that ``react-native-svg`` now has a way to accomplish it.

In the case of setting custom dimensions, we also don't seem to support that on the mobile app but we're confirming this with our devs.

**Comment by amyblais** [comment]
@hmhealey Do you know if we support setting custom dimensions on the mobile app?

https://community-release.mattermost.com/core/pl/ag1a7u8gwbf78bhwd3esgazmbw

**Comment by hmhealey** [comment]
No, we don't support custom dimensions on the mobile app. It's much more restrictive with what we can support for images.

**Comment by amyblais** [comment]
@dmarkdvt Would you be open to sharing these ideas on our feature request forum http://www.mattermost.org/feature-requests/?

**Comment by dmarkdvt** [comment]
Sure, I can do but isn't at least 4) a bug? The current mobile app should also ignore the dimension parameter =50 as it already does with =50x50px.

**Comment by amyblais** [comment]
I think so, posting the ticket for reference here: https://mattermost.atlassian.net/browse/MM-9746.

**Comment by dmarkdvt** [comment]
https://mattermost.uservoice.com/forums/306457-general/suggestions/39016390-mobile-app-should-support-custom-dimension-on-imag

**Comment by amyblais** [comment]
I'll close this as a duplicate for now as there is a Help Wanted issue open here: https://github.com/mattermost/mattermost-server/issues/13219. Let us know if you're interested in working on this issue!

**Comment by dmarkdvt** [comment]
> I'll close this as a duplicate for now as there is a Help Wanted issue open here: #13219. Let us know if you're interested in working on this issue!

Hmm, but the issue #13219 is another topic and this one is not integrated into that issue.

**Comment by lieut-data** [comment]
Thanks for reporting this issue and for the community discussion around it. After multiple years of inactivity, we're closing this to help keep our issue tracker focused on active work. If this is still relevant, please feel free to open a new issue with updated context and we'll be happy to take another look.
